master/internal/core_experiment.go

```diff
@@ -88,6 +88,29 @@ func (m *Master) getExperimentCheckpointsToGC(c echo.Context) (interface{}, erro
 		args.ExperimentID, args.ExperimentBest, args.TrialBest, args.TrialLatest)
 }
 
+func (m *Master) getExperimentModelFile(c echo.Context) error {
```

Comment:
since this is a new echo endpoint that is not going to be in grpc-gateway, and it doesn't produce automated docs, could we please add documentation using the method from PR #2529 here?

Reply:
Document has been added